test/e2e-common.sh

```diff
@@ -202,9 +202,15 @@ function install_head {
 function install_latest_release() {
   header ">> Installing Knative Eventing latest public release"
 
+  # Delete InMemoryController Webhook for downgrade 0.22 from 0.23.
```

Comment:
After 0.23 is cut, we remove this? (for downgrade to 0.23 from 0.24/main) ?

Reply:
yeah, but it does no harm